docker-for-mac/apple-m1.md

```diff
@@ -23,11 +23,14 @@ Click the following link to download the Apple M1 tech preview build:
 The tech preview of Docker Desktop for Apple M1 currently has the following limitations:
 
 - The tech preview build does not update automatically. You must manually install any future versions of Docker Desktop.
-- You must install Rosetta 2 as some binaries are still Darwin/AMD64. To install Rosetta 2 manually from the command line use this command:
+- You must install Rosetta 2 as some binaries are still Darwin/AMD64. To install Rosetta 2 manually from the command line, use this command:
+
     ```
     softwareupdate --install-rosetta
     ```
+
 - The HTTP proxy is not enabled.
+- Attempts to run Intel-based containers on Apple M1 machines can crash as QEMU sometimes fails to run the container. Therefore, we recommend that you run ARM64 containers on M1 machines as these containers are also faster when compared to Intel-based containers.
```

Comment:
This and the next line are a bit contradictory. Is there a way we can combine them, or at least acknowledge the tension (eg start whichever we put second with "However, ...")?

Reply:
The second sentence is a bit unwieldy. Suggest split into two: "... on M1 machines. These containers will also be faster and use less memory than Intel-based containers."